src/x86/sse.rs

```diff
@@ -51,7 +61,7 @@ pub fn _mm_movemask_ps(a: f32x4) -> i32 {
 }
 
 #[allow(improper_ctypes)]
-extern {
+extern "C" {
```

Comment:
What motivated this change?

Reply:
Looks like my rustfmt pass made this change. There is an option to turn this off. I'll do that.